frappe/public/js/frappe/ui/slides.js

```diff
@@ -338,12 +339,15 @@ frappe.ui.Slides = class Slides {
 		if (!this.unidirectional) this.bind_progress_dots();
 	}
 
-	make_prev_next_buttons() {
+	make_prev_next_complete_buttons() {
+		this.$footer.empty();
+
 		$(`<div class="row">
 			<div class="col-sm-4 text-left prev-div">
 				<button class="prev-btn btn btn-secondary btn-sm" tabindex="0">${__("Previous")}</button>
 			</div>
 			<div class="col-sm-8 text-right next-div">
+				<button class="complete-btn btn btn-sm primary">${__("Complete Setup")}</button>
 				<button class="next-btn btn btn-default btn-sm" tabindex="0">${__("Next")}</button>
```

Comment:
```suggestion
				<button class="prev-btn btn btn-secondary btn-sm" tabindex="0">${__("Previous", null, "Go to previous slide"))}</button>
			</div>
			<div class="col-sm-8 text-right next-div">
				<button class="complete-btn btn btn-sm primary">${__("Complete Setup", null, "Finish the setup wizard")}</button>
				<button class="next-btn btn btn-default btn-sm" tabindex="0">${__("Next"), null, "Go to next slide"}</button>
```

Let's add some [context](https://frappeframework.com/docs/v13/user/en/translations#6-adding-context-for-a-string) here.